outputs/.gitignore

```diff
@@ -1 +1,3 @@
 *.csv
+*.rds
```

Comment:
I don't know if we should ignore _all_ rds files.  They're used for package data as well as output.  Maybe just ignore them in the output directory?

Reply:
I think that does just ignore them in the output directory, right? Since the gitignore is there.